doc/architecture/tls_internals.rst

```diff
@@ -65,11 +65,7 @@ The main reasons why we moved to OpenSSL are:
 - We wanted TLS 1.3 support and MbedTLS doesn't have it.
 - We want to support QUIC, that doesn't work with MbedTLS.
```

Comment:
Shall we also drop all `QUIC`-related plans from this page too?

Reply:
Yes, the documentation should be plan-free. It's fine to explain why we did something at the time, and to describe what the code does.